xinference/client/restful/restful_client.py

```diff
@@ -676,6 +676,14 @@ def _check_cluster_authenticated(self):
             response_data = response.json()
             self._cluster_authed = bool(response_data["auth"])
 
+    def vllm_models(self) -> List[str]:
+        from ...model.llm.vllm.core import (
+            VLLM_SUPPORTED_CHAT_MODELS,
+            VLLM_SUPPORTED_MODELS,
+        )
+
+        return VLLM_SUPPORTED_MODELS + VLLM_SUPPORTED_CHAT_MODELS
```

Comment:
1. You cannot collect these global variables in the client code. You should send a http request to the fastapi server to get these information. Because in the other project, the client code is independent and only responsible for sending http requests. See: https://github.com/xorbitsai/inference-client
2. Couly you please show the chat and generate models supported by vllm separately?